integration_tests/suite/test_cdr.py

```diff
@@ -577,6 +612,42 @@ def test_given_no_call_logs_when_list_cdr_then_empty_list(self):
 
         assert_that(result, has_entries(items=empty(), filtered=0, total=0))
 
+    @multiple_call_logs(list_of_total_call_logs, batch_size=BULK_BATCH_SIZE)
+    def test_list_call_logs_when_large_number_of_users_and_calls(self):
```

Comment:
so you are saying that we do not need to write integration tests?

Reply:
Exactly, I would see this find of test in our load test scenarios but there's no HTTP queries being done in those tests at the moment. That's another task that we might tackle at the end of December but that should not block this ticket.

Reply:
Okay, so I will remove the test function but I recommend keeping the function `multiple_call_logs` that I implemented in `/fixtures` just in case we need it in the future :)